sample-project/app/controllers/users_controller.rb

```diff
@@ -54,7 +54,7 @@ def update
   # DELETE /users/1
   # DELETE /users/1.json
   def destroy
-    @user.destroy
+    @user.destroy #test
```

Comment:
```suggestion
    @user.destroy! # test
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>